gulpfile.js

```diff
@@ -86,15 +85,9 @@ const unminified3pTarget = 'dist.3p/current/integration.js';
 
 const maybeUpdatePackages = isTravisBuild() ? [] : ['update-packages'];
 
-// Used to start and stop the Closure nailgun server
-let nailgunRunnerReplacer;
-const nailgunRunner =
-    require.resolve('./third_party/nailgun/nailgun-runner');
-const nailgunServer =
-    require.resolve('./third_party/nailgun/nailgun-server.jar');
-const customRunner = require.resolve('./build-system/runner/dist/runner.jar');
-const NAILGUN_PORT = '2113'; // Also used in build-system/tasks/compile.js
-const NAILGUN_STARTUP_TIMEOUT_MS = 5 * 1000;
+// Also used in build-system/tasks/compile.js
+const CHECK_TYPES_NAILGUN_PORT = '2114';
+const DIST_NAILGUN_PORT = '2115';
```

Comment:
Why do we need two ports?

Reply:
So that in theory, it's possible to run `gulp dist`, `gulp check-types`, and `gulp nailgun-start` simultaneously in separate terminals.

Reply:
It wasn't necessary to use separate ports for `dist` and `check-types`. Restored to one port in #21953.